LoggerConfig/LoggerImplementation/AutoLog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Discord;
+using Serilog.Events;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuickEdit.LoggerConfig.LoggerImplementation;
+
+public class AutoLog
+{
+    // skipcq: CS-R1073
+    public static Task LogMessage(LogMessage message)
```

Comment:
`LogMessage()` sounds a bit generic but once again, I can't think of any other good name, so I think it's fine